checkups/kubevirt-vm-latency/vmlatency/internal/latency/pingparser.go

```diff
@@ -53,6 +54,10 @@ func ParsePingResults(pingResult string) (Results, error) {
 		err     error
 	)
 
+	if strings.Contains(pingResult, "100% packet loss") {
+		return Results{}, fmt.Errorf("%s: no connectivity - all packets were lost", errMessagePrefix)
```

Comment:
nit: I would stick to the original format: 
```suggestion
		return Results{}, fmt.Errorf("%s: no connectivity - 100% packet loss", errMessagePrefix)
```

Reply:
Done.